src/test/regress/citus_tests/query_generator/bin/citus_compare_dist_local_joins.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# make bash behave
+set -euo pipefail
+
+psql_user=$1
```

Comment:
what a database name?

Reply:
Database name is chosen as same with the username by psql. This is fine for running the generator both from test suite run on test cluster and also manual run on `citus_dev` since username matches database name.

Reply:
But I do not always use  `citus_dev`, it is also unusual to get the username/port but not the database name. I sometimes have different databases/users.

I think we should add the database name parameter as well